api-flask/app/caching.py

```diff
@@ -52,6 +52,14 @@ def cache_geojson(geojson, prefix):
     return cache_filepath
 
 
+def get_geojson_file(cached_filepath):
+    """Return geojson object as python dictionary."""
+    with open(cached_filepath, 'rb') as f:
+        data = json.load(f)
```

Comment:
`return json.load(f)` directly?

Reply:
Included in https://github.com/WFP-VAM/prism-frontend/pull/186/commits/932e4876ca33afccac1e7460bedb056b04c1ec6c